network/vif-route-qubes

```diff
@@ -120,9 +120,7 @@ case "$command" in
         ipcmd='add'
         iptables_cmd='-I PREROUTING 1'
         cmdprefix=''
-        ipv6_disabled=$(cat /proc/sys/net/ipv6/conf/"${vif}"/disable_ipv6 || echo 1)
-        # without a MAC address we will fail later with a confusing error
-        mac=$(xenstore-read "backend/vif/$domid/$sub/mac") || exit 1
```

Comment:
It is still used in `vif-qubes-nat.sh` script. I think this line is responsible for all fake_ip tests failures.